streaming/java/streaming-api/src/main/java/org/ray/streaming/api/function/impl/FilterFunction.java

```diff
@@ -14,7 +14,7 @@
   /**
    * The filter function that evaluates the predicate.
    *
-   * @param value The value to be filtered.
+   * @param value The VALUE to be filtered.
```

Comment:
It seems be modified unexpectedly, possibly in auto refactor by IDE. Please revert these unexpected changes.

Reply:
Thanks for replying. I'll check the codes.